hazelcast/src/main/java/com/hazelcast/cache/impl/merge/entry/KeyWithoutValueCacheEntryView.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2008-2018, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.cache.impl.merge.entry;
+
+import com.hazelcast.cache.CacheEntryView;
+import com.hazelcast.cache.impl.CacheDataSerializerHook;
+import com.hazelcast.nio.ObjectDataInput;
+import com.hazelcast.nio.ObjectDataOutput;
+import com.hazelcast.nio.serialization.Data;
+import com.hazelcast.nio.serialization.IdentifiedDataSerializable;
+
+import java.io.IOException;
+
+/**
+ * Entry view that only has {@code key} field set.
+ */
+public class KeyWithoutValueCacheEntryView
```

Comment:
What's the purpose of this class? I can't see where it is used.

Reply:
leftover, removed it.